ui-speedspacechart/src/__tests__/utils.spec.ts

```diff
@@ -94,6 +96,15 @@ describe('maxPositionValues', () => {
   });
 });
 
+describe('slopesValues', () => {
+  it('should return the correct minGradient and maxGradient', () => {
+    const { minGradient, maxGradient, slopesRange } = slopesValues(store);
+    expect(minGradient).toBe(-10);
+    expect(maxGradient).toBe(50);
+    expect(slopesRange).toBe(60);
+  });
+});
```

Comment:
Let's not forget to run the tests using npm run test to ensure we haven't broken anything. I have added one nominal case to the test.
```suggestion
  it('should return correct minGradient, maxGradient, slopesRange, and maxPosition', () => {
    const storeWithSlopes: Store = {
      ...store,
      slopes: [
        { gradient: 1, position: 10 },
        { gradient: 3, position: 20 },
        { gradient: 2, position: 15 },
        { gradient: 5, position: 25 },
      ],
    };

    const result = slopesValues(storeWithSlopes);

    expect(result).toEqual({
      minGradient: 1,
      maxGradient: 5,
      slopesRange: 4,
      maxPosition: 25,
    });
  });

  it('should handle empty slopes array', () => {
    const storeWithoutSlopes: Store = {
      ...store,
      slopes: [],
    };
    const result = slopesValues(storeWithoutSlopes);

    expect(result).toEqual({
      minGradient: Infinity,
      maxGradient: -Infinity,
      slopesRange: -Infinity,
      maxPosition: -Infinity,
    });
  });
```

Reply:
Thank you, done :)